### DISCUSSION TITLE
"hold for menu" button stays on screen during gameplay

### DISCUSSION THREAD

**Discussion by 165732** [ppy/osu#16669] [Q&A] [answered]
the "hold for menu" button stays on screen all the time during gameplay if the mouse haven't been moved at some point during the song, which is annoying when playing keyboard-based gamemodes like catch taiko or mania
https://streamable.com/m4913x

**Comment by peppy** [comment, marked as answer]
Just to confirm, is that because your cursor was hovering it before starting keyboard-only gameplay?

**Reply by 165732** [reply]
> Just to confirm, is that because your cursor was hovering it before starting keyboard-only gameplay?

no, my cursor was in the center of the screen
also the cursor don't have to hover over the button to make it disappear, moving the mouse in the middle of the screen will also work, pointer position doesn't matter

**Reply by peppy** [reply]
Thanks, I can reproduce the issue. As soon as you move the mouse it will fade, but if you never move the mouse it gets stuck there.

**Reply by peppy** [reply]
Should be resolved via #16678.

**Reply by 165732** [reply]
yep
also something unrelated to this issue, it would be nice to have the cursor disappear when it's idle in taiko catch and mania gamemodes, at the moment the cursor only disappears if it's on the playfield, in taiko for example if the cursor isn't on the conveyor belt where the notes are moving, it will never disappear
on stable the cursor never appear when playing catch or taiko

**Reply by peppy** [reply]
That one is tracked via https://github.com/ppy/osu/issues/2901. Requires a bit of thinking due to how we have the two different cursor types setup, but I think we'll get to it at some point.
